<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButtonBack">
        <property name="toolTip">
         <string>Poprzednia strona</string>
        </property>
        <property name="statusTip">
         <string>Przechodzi do poprzedniej  strony</string>
        </property>
        <property name="text">
         <string>&amp;Back</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/img/ButtonLeft.png</normaloff>:/img/img/ButtonLeft.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+B</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonForward">
        <property name="toolTip">
         <string>Następna strona</string>
        </property>
        <property name="statusTip">
         <string>Przechodzi do następnej strony</string>
        </property>
        <property name="text">
         <string>&amp;Forward</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/img/ButtonForward.png</normaloff>:/img/img/ButtonForward.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+F</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonReload">
        <property name="toolTip">
         <string>Przeładuj stronę</string>
        </property>
        <property name="statusTip">
         <string>Odświeża bieżącą stronę</string>
        </property>
        <property name="text">
         <string>&amp;Reload</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/img/ButtonReload.png</normaloff>:/img/img/ButtonReload.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWebView" name="webView">
      <property name="url">
       <url>
        <string>http://stackoverflow.com/questions/tagged/c%2B%2B</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Plik">
    <property name="title">
     <string>&amp;Plik</string>
    </property>
    <addaction name="action_Zako_cz_program"/>
   </widget>
   <widget class="QMenu" name="menuP_omoc">
    <property name="title">
     <string>Pomo&amp;c</string>
    </property>
    <addaction name="action_O_programie"/>
    <addaction name="actionO_Qt"/>
   </widget>
   <addaction name="menu_Plik"/>
   <addaction name="menuP_omoc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Zako_cz_program">
   <property name="text">
    <string>&amp;Zakończ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_O_programie">
   <property name="text">
    <string>&amp;O programie</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionO_Qt">
   <property name="text">
    <string>O &amp;Qt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
